### PR TITLE
15.0 mig pos cash move reason

### PR DESCRIPTION
**Pull request by ghost**
upgrade to V15, only minor changes

### CODE REVIEW

**Comment by legalsylvain** [comment]
hi @nikolausweingartmair . Thanks for porting this module.

could you take a look on tests that are failing ? 

thanks !

**Comment by ghost** [comment]
hi @legalsylvain,

I changed the tests because the status after initialization (open_session_cb()) changed from "open" to "opening_control".

File: pos_cash_move_reason/tests/test_pos_cash_move_reason.py [hunk 1]
```diff
@@ -71,8 +71,9 @@ def test_take_money(self):
     def test_take_invalid_amount(self):
         # Open New Session
         self.config.open_session_cb()
+        #
         session = self.PosSession.search(
```
Comment (legalsylvain) [inline pos_cash_move_reason/tests/test_pos_cash_move_reason.py:75]: Typo.

**Review by legalsylvain — APPROVED** [review]
Diff review. LGTM. Thanks !

File: pos_cash_move_reason/tests/test_pos_cash_move_reason.py (hunk 1 above)
Comment (ghost) [reply pos_cash_move_reason/tests/test_pos_cash_move_reason.py:75]: check

**Review by ivantodorovich — APPROVED** [review]

**Pull request merged**